lib/std/setutils.nim

```diff
@@ -0,0 +1,33 @@
+#
+#
+#           The Nim Compiler
+#        (c) Copyright 2020 Nim Contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## This module adds functionality to the builtin set
+## See also std/packedsets, setd/sets
+
+import typetraits
+
+type SetElement* = char|byte|bool|int16|uint16|enum|uint8|int8
+  ## The allowed types of a builtin set.
+
+template toSet*(iter: untyped): untyped =
+  ## Return a builtin set from the elements of iterable `iter`
+  ## Example: "ab".toSet == {'a','b'} 
```

Comment:
`Example: "ab".toSet == {'a','b'} ` is redundant, `runnableExamples:` will show usage